troi/patches/playlist_from_listenbrainz.py

```diff
@@ -0,0 +1,47 @@
+import click
+
+from troi import Playlist
+from troi.playlist import PlaylistFromJSPFElement
+import troi.musicbrainz.recording_lookup
+
+
+@click.group()
+def cli():
+    pass
+
+
+class ResavePatch(troi.patch.Patch):
```

Comment:
We may want to put this patch in LB or go some other way. Probably best to wait until troi is refactored to be a better library.